README.md

```diff
@@ -437,50 +437,68 @@ tabular_explainer = TabularExplainer(clf.steps[-1][1],
+
+![Visualization Dashboard Cohorts](./img/ModelPerformance.png)
+
+### Dataset explorer
+You can explore your dataset statistics by selecting different filters along the X, Y, and color axis of this tab to slice your data into different dimensions.
+
+![Visualization Dashboard Cohorts](./img/DatasetExplorer.png)
+
+The following plots provide a global view of the trained model along with its predictions and explanations.
+
+### Aggregate feature importance (global explanation)
+
+This view consists of two charts:
 |Plot|Description|
 |----|-----------|
-|Local Importance|Shows the top K (configurable K) important features for an individual prediction. Helps illustrate the local behavior of the underlying model on a specific data point.|
-|Perturbation Exploration (what if analysis)|Allows changes to feature values of the selected data point and observe resulting changes to prediction value.|
-|Individual Conditional Expectation (ICE)| Allows feature value changes from a minimum value to a maximum value. Helps illustrate how the data point's prediction changes when a feature changes.|
+|Feature Importance| Explore the top-k important features that impact your overall model predictions (a.k.a. global explanation). Use the slider to show descending feature importance values. Select up to three cohorts to see their feature importance values side by side.|
+|Dependence Plot|Click on any of the feature bars in the feature importance graph to see the relationship of the values of the selected feature to its corresponding feature importance values. Overall, this plot show how values of the selected feature impact model prediction.|
 
 
-![Visualization Dashboard Local Feature Importance](https://docs.microsoft.com/en-us/azure/machine-learning/service/media/machine-learning-interpretability-explainability/local-charts.png)
+![Visualization Dashboard Global](./img/GlobalExplanation.png)
 
-![Visualization Dashboard Feature Perturbation](https://docs.microsoft.com/en-us/azure/machine-learning/service/media/machine-learning-interpretability-explainability/perturbation.gif)
 
-![Visualization Dashboard ICE Plots](https://docs.microsoft.com/en-us/azure/machine-learning/service/media/machine-learning-interpretability-explainability/ice-plot.png)
 
+### Individual feature importance (local explanation) and what-if 
+You can click on any individual data point on the scatter plot to view its local feature importance values (local explanation) and individual conditional expectation (ICE) plot below. These are the capabilities covered in this tab:
 
-To load the visualization dashboard, use the following code:
 
-```python
-from interpret_community.widget import ExplanationDashboard
+|Plot|Description|
+|----|-----------|
+|Feature Importance Plot|Shows the top K (configurable K) important features for an individual prediction. Helps illustrate the local behavior of the underlying model on a specific data point.|
```

Comment:
Above, we use "top-k" rather than "top K". We should probably make that consistent